tests/unit/components/UI/Alert/DialogBox.spec.js

```diff
@@ -11,19 +15,66 @@ describe("DialogBox.vue", () => {
     const title = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        title,
+        title: title,
       },
     });
-    expect(wrapper.find("p").text()).toBe(title);
+
+    expect(wrapper.get('[data-test="title"]').text()).toBe(title);
   });
 
   it("renders description correctly", () => {
     const description = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        description,
+        description: description,
+      },
+    });
+
+    expect(wrapper.get('[data-test="description"]').text()).toBe(description);
+  });
+
+  it("renders image correctly", () => {
+    const wrapper = mount(DialogBox, {
+      props: {
+        iconConfig: {
+          enabled: true,
+          name: "check",
+          class: "w-12 h-12",
+        },
+      },
+      global: {
+        stubs: {
+          InlineSvg: InlineSvg,
+        },
       },
     });
-    expect(wrapper.find("p").text()).toBe(description);
+    expect(wrapper.get('[data-test="icon"]').html()).toContain("img");
+    expect(wrapper.get('[data-test="icon"]').classes()).toEqual(
```

Comment:
just curious, how is it different from `.toContain()`?

Reply:
I'm guessing to contain will pass even if it's not an exact match and toEqual will have to be an exact match

Reply:
`toContain` won't work when the returned value is an array. So, if the actual value was `['a', 'b', 'c']`, `.toContain('a')` won't work.